clang/test/Sema/gh106576.c

```diff
@@ -0,0 +1,16 @@
+// RUN: %clang_cc1 -fsyntax-only -verify %s
+
+typedef _Atomic char atomic_char;
+
+typedef _Atomic char atomic_char;
```

Comment:
Presumably the redundant typedef is not a critical part of this test.

Reply:
It isn't, I have removed the duplicated line.